webapp/client/src/assets/images/list_marker.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="27" height="16" viewBox="0 -10 27 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think this should go into `assets/icons` not `assets/images`. No need to replicate the mistakes of old… :D